### PR TITLE
Add "pt" language to LINGUAS and Portuguese translation .po file in extra folder

### PR DESCRIPTION
**Pull request by hugok79**
Updates for Portuguese(pt) language

### CODE REVIEW

File: po/extra/pt.po [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+msgid "Fix wrong icon displayed in the notifications"
+msgstr "Corrige ícone incorreto apresentado nas notificações"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:178
+msgid "Fix Desktop file being invalid"
+msgstr "Corrige o ficheiro do ambiente de trabalho de ser inválido"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:179
+msgid "Bump granite dependency version"
+msgstr "Impacto da versão de dependência do granite"
```
Comment (waldyrious) [inline po/extra/pt.po:328]: "Bump" doesn't mean a literal collision in this context. It's jargon for a version upgrade.

File: po/extra/pt.po [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+msgstr "Pequenas correções"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:161
+msgid "Added an ability to open packages with Eddy in the file manager"
+msgstr ""
+"Adicionada a capacidade de abrir pacotes com o Eddy no gestor de ficheiros"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:162
+msgid "Show a spinner when loading packages"
+msgstr "Mostrar um botão giratório ao carregar pacotes"
```
Comment (waldyrious) [inline po/extra/pt.po:307]: Shouldn't this be "Mostra", for verb tense consistency with the other entries?

File: po/extra/pt.po [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+msgstr ""
+"Carregue todos os pacotes automaticamente a partir da sua pasta de "
+"Transferências"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:16
+msgid ""
+"Complete system integration including notifications and progress bar in the "
+"dock"
+msgstr ""
+"Integração completa do sistema incluindo notificações e barra de progresso "
```
Comment (waldyrious) [inline po/extra/pt.po:57]: I believe "system integration" would be better translated as "integração com o sistema", as it refers to the underlying operating system.

File: po/extra/pt.po [hunk 4]
```diff
@@ -0,0 +1,360 @@
...
+msgstr "Instala vários pacotes ao mesmo tempo"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:14
+msgid "Upgrade and downgrade existing packages"
+msgstr "Atualiza e desatualiza de pacotes existentes"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:15
+msgid "Load all packages automatically from your Downloads folder"
+msgstr ""
+"Carregue todos os pacotes automaticamente a partir da sua pasta de "
```
Comment (waldyrious) [inline po/extra/pt.po:49]: For the verb tenses to be consistent, this should say "Carrega".

File: po/extra/pt.po (hunk 1 above)
Comment (hugok79) [reply po/extra/pt.po:328]: Yes I know. I don't usually translate these types of words but I always choose to leave the translations without anglicisms(as much Portuguese as possible). Do you have any suggestions for this string?

File: po/extra/pt.po (hunk 2 above)
Comment (hugok79) [reply po/extra/pt.po:307]: I try to make the translations as formal as possible but in this case I believe that @waldyrious is right.
I'll fix that.

File: po/extra/pt.po (hunk 3 above)
Comment (hugok79) [reply po/extra/pt.po:57]: Thanks for the tip.
I'll fix that.

File: po/extra/pt.po (hunk 4 above)
Comment (hugok79) [reply po/extra/pt.po:49]: Thanks for the tip.
I'll fix that.

File: po/extra/pt.po (hunk 1 above)
Comment (waldyrious) [reply po/extra/pt.po]: I suppose "Atualização", or something to that effect, would work.

File: po/extra/pt.po [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+msgid "Features include:"
+msgstr "Funcionalidades incluídas:"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:13
+msgid "Install multiple packages at once"
+msgstr "Instala vários pacotes ao mesmo tempo"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:14
+msgid "Upgrade and downgrade existing packages"
+msgstr "Atualiza e desatualiza de pacotes existentes"
```
Comment (waldyrious) [inline po/extra/pt.po:44]:
```suggestion
msgstr "Atualiza e desatualiza pacotes existentes"
```

File: po/extra/pt.po [hunk 6]
```diff
@@ -0,0 +1,359 @@
...
+msgid ""
+"This release fixes small annoyances with the UI and includes new "
+"translations:"
+msgstr ""
+"Esta versão corrige pequenos constrangimentos com a IU e inclui novas "
+"traduções:"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:64
+msgid "Brazilian Portuguese translation (Paulo Galardi)"
+msgstr "Tradução do Português do Brasil (Paulo Galardi)"
```
Comment (waldyrious) [inline po/extra/pt.po:141]:
```suggestion
msgstr "Tradução para Português do Brasil (Paulo Galardi)"
```

File: po/extra/pt.po [hunk 7]
```diff
@@ -0,0 +1,359 @@
...
+msgid "French translation (nvivant)"
+msgstr "Tradução francesa (nvivant)"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:71
+msgid ""
+"Fast switching to the detailed view while loading packages caused "
+"unintentional auto-switching to list view"
+msgstr ""
+"Mudança rápida para a vista detalhada durante o carregamento de pacotes "
+"causou a comutação automática não intencional para a vista de lista"
```
Comment (waldyrious) [inline po/extra/pt.po:161]: Perhaps this is a little clearer?

```suggestion
"causou inadvertidamente a alteração automática para a vista de lista"
```

File: po/extra/pt.po [hunk 8]
```diff
@@ -0,0 +1,359 @@
...
+#: data/com.github.donadigo.eddy.appdata.xml.in:84
+msgid "Replace confusing quit dialog actions with more explicit ones"
+msgstr ""
+"Substitui as ações confusas da caixa de diálogo de saída por outras mais "
+"explícitas"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:85
+#: data/com.github.donadigo.eddy.appdata.xml.in:98
+msgid "Update application metadata"
+msgstr "Atualizar os metadados da aplicação"
```
Comment (waldyrious) [inline po/extra/pt.po:201]: For consistency, this should be either "Atualiza os..." or "Atualização dos..."

File: po/extra/pt.po [hunk 9]
```diff
@@ -0,0 +1,359 @@
...
+msgid "Update Dutch and German translations"
+msgstr "Atualização de traduções em holandês e alemão"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:94
+msgid "\"About Eddy\" now opens this AppCenter page"
+msgstr "\"Sobre o Eddy\" abre agora esta página da Loja de Aplicações"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:105
+msgid "Fix about dialog not showing correctly"
+msgstr "Correção sobre a caixa de diálogo não ser apresentada corretamente"
```
Comment (waldyrious) [inline po/extra/pt.po:213]: I believe "about" in the English line refers to the About Eddy dialog box.

File: po/extra/pt.po [hunk 10]
```diff
@@ -0,0 +1,359 @@
...
+msgid "Display the progress of current operation in the dock"
+msgstr "Mostrar o progresso da operação atual na Doca"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:138
+msgid "Sort packages by their state and name in the list view"
+msgstr "Ordena os pacotes por estado e nome na visualização de lista"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:145
+msgid "Save window state between sessions"
+msgstr "Guardar o estado da janela entre as sessões"
```
Comment (waldyrious) [inline po/extra/pt.po:284]: Verb tense consistency:

```suggestion
msgstr "Guarda o estado da janela entre as sessões"
```

File: po/extra/pt.po [hunk 11]
```diff
@@ -0,0 +1,359 @@
...
+msgid "Ctrl-Q now exists the app"
+msgstr "Ctrl-Q agora faz sair da aplicação"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:136
+msgid "Do not display error dialog when operation cancelled"
+msgstr "Não mostra a caixa de diálogo de erro quando a operação é cancelada"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:137
+msgid "Display the progress of current operation in the dock"
+msgstr "Mostrar o progresso da operação atual na Doca"
```
Comment (waldyrious) [inline po/extra/pt.po:276]: Verb tense consistency:

```suggestion
msgstr "Mostra o progresso da operação atual na Doca"
```

File: po/extra/pt.po [hunk 12]
```diff
@@ -0,0 +1,359 @@
...
+msgid "Fix Desktop file being invalid"
+msgstr "Corrige o ficheiro do ambiente de trabalho de ser inválido"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:179
+msgid "Bump granite dependency version"
+msgstr "Atualização da versão de dependências do granite"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:180
+msgid "Update debian packaging"
+msgstr "Atualiza a empacotagem debian"
```
Comment (waldyrious) [inline po/extra/pt.po:331]: I'd say Debian should be capitalized here, as it is a proper noun. Although the English original doesn't do this in this particular line, it does in line 346 below (referring to the source location `data/com.github.donadigo.eddy.desktop.in:9`).

File: po/extra/pt.po [hunk 13]
```diff
@@ -0,0 +1,359 @@
...
+msgid "Eddy"
+msgstr "Eddy"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:8
+msgid "Install and Manage .deb Files"
+msgstr "Instalar e gerir ficheiros .deb"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:10
+msgid "Install, uninstall and view information about debian packages."
+msgstr "Instalar, desinstalar e ver informação sobre pacotes debian."
```
Comment (waldyrious) [inline po/extra/pt.po:32]: Capitalization, as suggested in [another comment](https://github.com/donadigo/eddy/pull/96/files#r551283476):

```suggestion
msgstr "Instalar, desinstalar e ver informação sobre pacotes Debian."
```

File: po/extra/pt.po (hunk 5 above)
Comment (hugok79) [reply po/extra/pt.po:44]: Fixed

File: po/extra/pt.po (hunk 6 above)
Comment (hugok79) [reply po/extra/pt.po:141]: Fixed

File: po/extra/pt.po (hunk 7 above)
Comment (hugok79) [reply po/extra/pt.po:161]: Fixed

File: po/extra/pt.po (hunk 8 above)
Comment (hugok79) [reply po/extra/pt.po:201]: Fixed

File: po/extra/pt.po (hunk 9 above)
Comment (hugok79) [reply po/extra/pt.po:213]: Agree. Thanks for the tip.
Fixed

File: po/extra/pt.po (hunk 10 above)
Comment (hugok79) [reply po/extra/pt.po:284]: Fixed

File: po/extra/pt.po (hunk 11 above)
Comment (hugok79) [reply po/extra/pt.po:276]: Fixed

File: po/extra/pt.po (hunk 12 above)
Comment (hugok79) [reply po/extra/pt.po:331]: Agree. All Debian strings should be capitalized.
Fixed

File: po/extra/pt.po (hunk 13 above)
Comment (hugok79) [reply po/extra/pt.po:32]: Fixed

**Comment by hugok79** [comment]
Thanks for the help @waldyrious 
I am also the Portuguese translator of the elementary OS and I think the only one...!
I have kept the translation 100% completed
 If you can take a look at my translation here https://l10n.elementary.io/languages/pt/
All help is appreciated...

File: po/extra/pt.po [hunk 14]
```diff
@@ -0,0 +1,361 @@
...
+msgstr "Atualização de traduções em holandês e alemão"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:94
+msgid "\"About Eddy\" now opens this AppCenter page"
+msgstr "\"Sobre o Eddy\" abre agora esta página da Loja de Aplicações"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:105
+msgid "Fix about dialog not showing correctly"
+msgstr ""
+"Correção da caixa de diálogo \"Sobre o Eddy\" de não ser apresentada "
```
Comment (waldyrious) [inline po/extra/pt.po:214]:
```suggestion
"Correção da caixa de diálogo \"Sobre o Eddy\" não ser apresentada "
```

**Comment by waldyrious** [comment]
> Thanks for the help @waldyrious

Happy to help! Thank _you_ for doing most of the work :)

> I am also the Portuguese translator of the elementary OS and I think the only one...!
> I have kept the translation 100% completed

Amazing! Excellent work! 👏

> If you can take a look at my translation here l10n.elementary.io/languages/pt
> All help is appreciated...

I actually did register to help out with the Portuguese translation back when it was handled in Launchpad, but never had the bandwidth to really dedicate to that work. I even registered my account in the new translation site (https://l10n.elementary.io/user/waldyrious/) but did just a few translations while exploring the platform.

What I settled into was just watching the repositories of some of my favorite apps (such as Eddy), and helping out here and there however I can (with translation, documentation, review, etc.). So I am sorry to say I won't likely be able to review all your work from top to bottom.

That said, if there are particular strings you are in doubt about, feel free to ping me anytime and I'll gladly try to help out!

File: po/extra/pt.po (hunk 14 above)
Comment (hugok79) [reply po/extra/pt.po:214]: I will accept the suggestion.
Thank you once more.

**Review by donadigo — APPROVED** [review]
Thank you for your work!

**Pull request merged**